analyzer-comments/javascript/annalyns-infiltration/can_simplify_conditionals.md

```diff
@@ -0,0 +1,4 @@
+Simplify or remove if statements.
+
+This exercise is meant to explore booleans and boolean operators.
```

Comment:
The word "Boolean" is capitalized, even the British National Corpus agrees.

Reply:
I agree too, but in JS documentation, `Boolean` is the function and `boolean` is the type (see: [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean))